presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -83,4 +83,15 @@ public void testShowColumnStats()
 
         assertEquals(result, expectedStatistics);
     }
+
+    @Test
+    public void testDecimal()
```

Comment:
Tests in `TestSqlParser` turned out to be not sufficient, as `Decimals` needed update too. Thus I added tests in `TestDecimals` but also here